add a new imex integrator

wisYue opened the pull request.

jmstone commented:
Thanks, Yue.  I've created a new branch in which I implemented the C-shock test, and added this integrator to that branch, so it will get added to main when I merge that branch.  So this PR is now not needed.

I'm afraid this new integrator does not pass the C-shock test, so we will have to look into this further before using it in applications.

wisYue commented:
Hi Jim,
  

  
What is the issue of the C-shock test? I will back to IAS next Monday, so probably we can talk more about this.
  

  
Best,
  
Yue
  
  
  
  
  
  
  

  
  
>   
> On Sep 18, 2024 at 6:46 AM,  <jmstone ***@***.***)>  wrote:
>   
>   
>   
>
>
>
>   
>
> Thanks, Yue. I've created a new branch in which I implemented the C-shock test, and added this integrator to that branch, so it will get added to main when I merge that branch. So this PR is now not needed.
>
>   
>
> I'm afraid this new integrator does not pass the C-shock test, so we will have to look into this further before using it in applications.
>
>   
>
> —
>  Reply to this email directly,   view it on GitHub (https://github.com/IAS-Astrophysics/athenak/pull/586#issuecomment-2358520264), or   unsubscribe (https://github.com/notifications/unsubscribe-auth/AKN56AO5IFDLDVJYRDDWOQTZXF74HAVCNFSM6AAAAABNZPFHCCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGNJYGUZDAMRWGQ).
>  You are receiving this because you authored the thread.Message ID:  ***@***.***>
>
>   
>
